### PR TITLE
Improve version number

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by Megamouse** [comment]
care to add the regex stuff or sth similar like what I smuggled into bakuretsu?

**Comment by Nekotekina** [comment]
Should fix problems with Celerons

**Comment by Nekotekina** [comment]
May fix #2950

**Comment by Xcedf** [comment]
Cpu-z tells me that my 6700k has TSX, yet i have 
F {MFC Thread} MEM: Access violation reading location 0x0 
in every game i'm trying to run

**Comment by AniLeo** [comment]
Wait for newer build. Diva F works though on the first one

**Comment by Luffykun007** [comment]
Performance has improved a bit here whith i5 6400

File: rpcs3/Emu/Cell/SPUAnalyser.cpp [hunk 1]
```diff
@@ -328,7 +328,7 @@ std::shared_ptr<spu_function_t> SPUDatabase::analyse(const be_t<u32>* ls, u32 en
 		m_db.emplace(key, func);
 	}
 
-	LOG_SUCCESS(SPU, "Function detected [0x%05x-0x%05x] (size=0x%x)", func->addr, func->addr + func->size, func->size);
+	LOG_NOTICE(SPU, "Function detected [0x%05x-0x%05x] (size=0x%x)", func->addr, func->addr + func->size, func->size);
```
Comment (AniLeo) [inline rpcs3/Emu/Cell/SPUAnalyser.cpp]: T h a n k 
Y o u

**Comment by Xcedf** [comment]
Performance has improved, once again great work

**Pull request merged**